src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/life_time.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_plugin/life_time.hpp"
+
+using namespace ov::test::behavior;
+namespace {
+    INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, OVHoldersTest,
+            ::testing::Values(CommonTestUtils::DEVICE_GPU),
+            OVHoldersTest::getTestCaseName);
+
+    INSTANTIATE_TEST_SUITE_P(smoke_VirtualPlugin_BehaviorTests, OVHoldersTest,
+                    ::testing::Values("AUTO:GPU",
+                                        "MULTI:GPU",
+                                        //CommonTestUtils::DEVICE_BATCH,
+                                        "HETERO:GPU"),
+            OVHoldersTest::getTestCaseName);
```

Comment:
I suppose in these tests we don't test the case when the `Tensor` is removed last. E.g.
```c++
ov::Tensor t = ov::Core().compile_model(m, "AUTO:GPU").create_infer_request().get_input_tensor();
```
Can we add such use case?

Reply:
ahh did you mean that tensor is destroyed after compiled model, plugin and ie code? If yes, I think this is covered below and here I'm instantiating these tests for AUTO/MULTI/HETERO:
https://github.com/songbell/openvino/blob/5f3cadcfc4977665d8cf5682fcdda269d15a5651/src/tests/functional/plugin/shared/src/behavior/ov_plugin/life_time.cpp#L76

Reply:
right, thanks